src/components/ProfileValidation/ProfileValidation.tsx

```diff
@@ -232,7 +232,8 @@ export enum ValidationWizardStep {
 
 function BodyControlFactory(args: {
   step: ValidationWizardStep
-  onFormChange: (a: boolean) => void
+
```

Comment:
Tacking a couple of issues here:
 - with the current architecture the enabled/disabled state of the continue button did not persist during 'back' navigation
 - verification submission did not get updated.
 Now 'continue/submit' button is it's own dumb component which gets included into the individual subcontrols